src/app/components/FrostedGlassPromo/lazy.jsx

```diff
@@ -1,8 +1,11 @@
+import React from 'react';
 import loadable from '@loadable/component';
 
-export default loadable(() =>
-  import(
-    /* webpackChunkName: "frosted_promo" */
-    '.'
-  ),
+export default loadable(
+  () =>
+    import(
+      /* webpackChunkName: "frosted_promo" */
+      '.'
+    ),
+  { fallback: <span data-testid="frosted-promo-loader" /> },
```

Comment:
This looks cool. So the browser doesn't load the component chunk outlined until it comes into view?

Reply:
Ah I'm not sure Loadable cares about scroll position - but it does exclude the code from the bundle if the component is not actually invoked at runtime though.

eg, without loadable, this component and it's subtree (which is quite big because of the colour picker) would need included in the bundle, even if the component was toggled off